centaur/src/main/scala/centaur/test/CentaurTestException.scala

```diff
@@ -15,6 +15,7 @@ import cromwell.api.model.{SubmittedWorkflow, WorkflowMetadata}
 case class CentaurTestException private(message: String,
                                         testName: String,
                                         workflowIdOption: Option[String],
```

Comment:
Could `workflowIdOption: Option[String]` be changed to `workflowIdList: List[String]`? I think `tryTryAgain` wouldn't have to change much... just not sure what to replace `getOrElse` with.

Reply:
We could do that, but we'll still need to know which is the primary workflow that failed, since we use that in various places (like logs). If this pattern is expanded to cover all cases where we need to clear cache for multiple workflows, we'll end up with "associated workflows" in many cases where they aren't directly related to the error, and thus aren't useful to tell users about. 

I may end up changing `associatedWorkflowIdOptions` to a list, though, since I noticed we have at least one test format that runs three workflows.